cmake/utils.cmake

```diff
@@ -123,6 +123,11 @@ function (get_torch_gpu_compiler_flags OUT_GPU_FLAGS GPU_LANG)
       "-U__HIP_NO_HALF_CONVERSIONS__"
       "-U__HIP_NO_HALF_OPERATORS__"
       "-fno-gpu-rdc")
+      
+    string(TOUPPER ${CMAKE_BUILD_TYPE} VLLM_BUILD_TYPE)
```

Comment:
set(CMAKE_HIP_FLAGS_DEBUG "${CMAKE_HIP_FLAGS_DEBUG} -O0 -ggdb3")
set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -O0 -ggdb3")
probably cleaner

Reply:
sorry, but I don't see where FLAGS_DEBUG are used now? Or you mean promote those flags into other places in CMakeLists.txt?

Reply:
It's a CMake feature
https://cmake.org/cmake/help/v3.8/variable/CMAKE_LANG_FLAGS_DEBUG.html